api/db-cdr/generate-cdr/make-bq-public-data.sh

```diff
@@ -1,19 +1,17 @@
 #!/bin/bash
 
-# This copies the bq cdr counts dataset to public and runs some updates
+# # This generates big query count databases public that get put in cloudsql for data browser
```

Comment:
Could you add some notes there about the preconditions for this script? Namely that we're now requiring that we have a raw form of the count tables already present in the given public BQ dataset (and this script will modify those tables in place).

Reply:
Done.